specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/ManagedEnvironments.json

```diff
@@ -1058,6 +1058,101 @@
         }
       }
     },
+    "AppInsightsConfiguration": {
+      "description": "Configuration of Application Insights ",
+      "type": "object",
+      "properties": {
+        "connectionString": {
```

Comment:
It seems to be a secret, would you please add `x-ms-secret: true`?

refs: https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret